CHANGES

```diff
@@ -1,6 +1,14 @@
 Jinja2 Changelog
 ================
 
+Version 2.9.6
+-------------
+(bugfix release, released on February ???? 2017)
```

Comment:
Why put this into a bugfix release though? This is not acceptable according to semver.